server/common/variables.js

```diff
@@ -1,5 +1,9 @@
 const DB_URL = process.env.DB_URL || 'mongodb://localhost:27017/eoc';
 const ENDPOINT_URL = 'http://localhost:8080';
 const FRONTEND_URL = 'http://localhost:3000';
+const ListType = Object.freeze({
+  LIMITED: 'limited',
+  SHARED: 'shared'
+});
```

Comment:
ListType is defined in list module in index.jsx. But this types are needed on back-end also, so I put it in variables.js file because on back-end we don't have analogous place...

Reply:
Please don't close conversation where you have responded to my notice